{"name": "test_running_test", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable", "trace": "def test_running_test():\n    #driver.background_app(5)\n      driver.find_element(by=AppiumBy.ID,value='com.android.chrome:id/home_button').click()\n      time.sleep(3)\n      el=driver.find_element(by=AppiumBy.ID, value=\"com.android.chrome:id/search_box_text\")\n      el.send_keys('https://qa.telli.cc/consumer/oauth2/login?ReturnUrl=%2Fconsumer%2Foauth2%2Fconnect%2Fauthorize%3Fclient_id%3Dauthorization-code-client%26redirect_uri%3Dhttps%253A%252F%252Fqa.telli.cc%252Fconsumer%252Fmember%252Fcallback%26response_type%3Dcode%26scope%3D%2520%26state%3D3e390076cc644b3788ed41ed39f2d69d%26code_challenge%3DIlR89jtBkm3IK3MQ4GEat-rfLQAIfsyZ8pRaw_nYbkg%26code_challenge_method%3DS256%26response_mode%3Dquery%26companyId%3D97d288ac-5f17-11ed-afa6-00ffaf2156c9')\n      time.sleep(3)\n      popup=driver.get_screenshot_as_png()\n      allure.attach(popup, '確認進入SSO登入頁', allure.attachment_type.PNG)\n      driver.find_element(by=AppiumBy.ID, value=\"com.android.chrome:id/line_1\").click()\n>     time(3)\nE     TypeError: 'module' object is not callable\n\napp_test.py:43: TypeError"}, "attachments": [{"name": "確認進入SSO登入頁", "source": "0009491d-fdd1-4ab9-acbc-9dff2a7a95aa-attachment.png", "type": "image/png"}], "start": 1669622462308, "stop": 1669622470520, "uuid": "003600bf-9f8f-449f-9427-4d21f6cb5af7", "historyId": "37e2e867df4d816d3f0af45042b7e416", "testCaseId": "540b3e91c86314649350359c7cd0215f", "fullName": "app_test#test_running_test", "labels": [{"name": "suite", "value": "app_test"}, {"name": "host", "value": "C010700042"}, {"name": "thread", "value": "25584-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "app_test"}]}